### DISCUSSION TITLE
The volume OSD cannot hide

### DISCUSSION THREAD

**Discussion by wklhtcone** [MonitorControl/MonitorControl#813] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

My monitor is LG UK850, I turn on the "hide macOS volume OSD" function but it doesn't wo
![IMG_3462](https://user-images.githubusercontent.com/24488134/143599577-d4803b43-994a-4a67-84da-e92fd9b0adcd.jpg)
rk

### Expected behavior

fix this issue so that I can hide the volume OSD

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.0.1 Monterey
- Mac model: MacBook Pro (15-inch, 2019)
- MonitorControl version: v4.0.2(6965)
- Monitor(s): LG 27UK850
- Monitor Cable(s)/Connection(s): USB-C to USB-C
- Apple Silicon/M1 (yes or no):no
```

**Comment by waydabber** [comment, marked as answer]
Hi, some displays show their own OSD for volume change. There is no function currently in the app to remove the display's OSD. You can hide the macOS volume OSD only.
